test/EndToEnd/GivenWindows50App.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+
+namespace EndToEnd
+{
+    public class GivenWindows50App : TestBase
+    {
+        [WindowsOnlyFact]
+        public void ItCanBuildAndRun()
```

Comment:
If possible, it would be good to have this app call some sort of WinRT API to verify that the references are being brought in correctly.

Reply:
Use updated to use a json API and assert the output